sideboard/config.py

```diff
@@ -9,14 +10,72 @@
 from validate import Validator
 
 
+_defaults_re = re.compile(r'(.+)-defaults(\.\w+)$')
+
+
+def uniquify(x):
+    """
+    Returns an ordered copy of `x` with duplicate items removed.
+
+    >>> uniquify(['a', 'b', 'a', 'c', 'a', 'd', 'a', 'e'])
+    ['a', 'b', 'c', 'd', 'e']
+
+    """
+    seen = set()
+    return [i for i in x if i not in seen and not seen.add(i)]
```

Comment:
I'm actually a fan of list comprehension, so I don't think you need to explode it. It's pretty clear what this function does -- I just wanted to know exactly how it was working. It would be different if this was part of a more complex function, but because it's in its own function it is totally fine as-is.